servicetalk-transport-netty-internal/src/main/java/io/servicetalk/transport/netty/internal/NettyIoExecutors.java

```diff
@@ -101,6 +111,17 @@ public static NettyIoExecutor fromNettyEventLoop(EventLoop eventLoop) {
         return new EventLoopIoExecutor(eventLoop, true);
     }
 
+    /**
+     * Creates a new instance of {@link NettyIoExecutor} using the passed {@link EventLoop}.
+     *
+     * @param eventLoop {@link EventLoop} to use to create a new {@link NettyIoExecutor}.
+     * @param isIoThreadSupported if {@code true} then event loop threads are marked as {@link IoThreadFactory.IoThread}
+     * @return New {@link NettyIoExecutor} using the passed {@link EventLoop}.
+     */
+    public static NettyIoExecutor fromNettyEventLoop(EventLoop eventLoop, boolean isIoThreadSupported) {
```

Comment:
Consider adding a similar overload for `fromNettyEventLoopGroup`, in case someone uses it to convert `EventLoopGroup` to `IoExecutor`